client/src/components/SearchBox.js

```diff
@@ -2,6 +2,9 @@
 import React from 'react';
 import Form from 'react-bootstrap/Form';
 import Col from 'react-bootstrap/Col';
+import { BackendClient } from '../requests';
+
+let conn = new BackendClient();
```

Comment:
**conn**, someone recently said to me "are you charged by chars?!", 
**connection** looks better :)